frontend/packages/knative-plugin/src/components/overview/__tests__/EventSinkServicesOverviewList.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as utils from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink } from '@console/internal/components/utils';
+import { ServiceModel } from '@console/knative-plugin';
+import EventSinkServicesOverviewList from '../EventSinkServicesOverviewList';
+import { mockEventSourceData } from '../__mocks__/overview-knative-mock';
+
+describe('EventSinkServicesOverviewList', () => {
+  it('should component exists', () => {
+    const wrapper = shallow(<EventSinkServicesOverviewList obj={mockEventSourceData} />);
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should have title Knative Services', () => {
+    const wrapper = shallow(<EventSinkServicesOverviewList obj={mockEventSourceData} />);
+    expect(wrapper.find(utils.SidebarSectionHeading)).toHaveLength(1);
+    expect(
+      wrapper
+        .find(utils.SidebarSectionHeading)
+        .at(0)
+        .props().text,
+    ).toEqual('Knative Services');
+  });
```

Comment:
This is simply asserting that the static markup is returned.
I don't like tests like this because it needs to be kept in sync at all times. Chances are there will never be a breakage in this test unless we explicitly want to change the markup in which case it's a 1:1 change between the markup and the test itself.
cc @rohitkrai03  @andrewballantyne